packages/backend/src/apps/formsg/triggers/new-submission/get-data-out-metadata.ts

```diff
@@ -123,6 +123,78 @@ function buildAnswerArrayMetadatum(
   }
 }
 
+function buildVerifiedSubmitterInfoMetadata(
+  data: IJSONObject,
+): IDataOutMetadata | null {
+  if (!data.verifiedSubmitterInfo) {
+    return null
+  }
+
+  const metadata: IDataOutMetadata = Object.create(null)
+
+  for (const key of Object.keys(data.verifiedSubmitterInfo)) {
+    switch (key) {
+      case 'uinFin':
+        metadata.uinFin = { label: 'NRIC/FIN (Verified)' }
+        break
+      case 'sgidUinFin':
+        metadata.sgidUinFin = { label: 'NRIC/FIN (Verified)' }
+        break
+      case 'cpUid':
+        metadata.cpUid = { label: 'CorpPass UID (Verified)' }
+        break
+      case 'cpUen':
+        metadata.cpUen = { label: 'CorpPass UEN (Verified)' }
```

Comment:
Do u think we should include in the label this is **payer's** UIN and UEN?

Reply:
Hmm, this field may be available in non payment forms so cannot reference payer. I feel like we can call it "form submitter verified NRIC" or something though.

I grab stacey for a copy review in another PR. Going to land this first to enable MOE use case.